ykllvmwrap/src/ykllvmwrap.cc

```diff
@@ -72,6 +72,29 @@ std::unique_ptr<Module> load_module(LLVMContext &Context, char *Ptr,
   return M;
 }
 
+std::vector<Value*> get_trace_inputs(Function *F, uintptr_t BBIdx) {
+  std::vector<Value*> Vec;
+  auto It = F->begin();
+  // Skip to the first block in the trace which contains the `start_tracing`
+  // call.
+  std::advance(It, BBIdx);
+  BasicBlock *BB = &*It;
+  for (auto I = BB->begin(); I != BB->end(); I++) {
+    if (isa<CallInst>(I)) {
+      CallInst *CI = cast<CallInst>(&*I);
+      if (CI->getCalledFunction()->getName() == YKTRACE_START) {
+          auto Arg = CI->arg_begin();
```

Comment:
We seem to have moved from 2-indent to 4-indent here?

Reply:
I'll do a formatting path after squashing if that's ok. That'll make my life easier.

Reply:
If you mean "after everything else but before squashing", that's fine ;)